nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -753,8 +760,8 @@ in
 
         serviceConfig = {
           ExecStart = "@${pkg}/bin/httpd httpd -f ${httpdConf}";
-          ExecStop = "${pkg}/bin/httpd -f ${httpdConf} -k graceful-stop";
-          ExecReload = "${pkg}/bin/httpd -f ${httpdConf} -k graceful";
```

Comment:
Why are you using `apachectl` (a wrapper around `httpd`) instead?

Reply:
:confused: So, I tested this command line before I made the change because I thought it was a typo, and it didn't work (invalid arguments error)... But I just tested it again and it worked. I'll revert. Thanks for highlighting

Reply:
Thanks!